src/labels/CustomLabels.labels

```diff
@@ -2216,12 +2232,20 @@
         <value>Select installments from the list below. You can pause for up to 12 consecutive installments.</value>
     </labels>
     <labels>
-        <fullName>RD2_PauseHeader</fullName>
+        <fullName>RD2_PausePermissionRequired</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>User does not have a permission to edit or create Recurring Donations</shortDescription>
+        <value>You don&apos;t have permissions to Pause Recurring Donations. Please contact your system administrator for create and edit access on Recurring Donation.</value>
```

Comment:
@sam-knox , can you please verify the new custom label text displayed when user does not have read, edit and create permissions on the Recurring Donation SObject? Also, there is another new `lblInsufficientPermissions` and I added a "."  for the `RD2_PauseSaveSuccessMessage` and `RD2_PauseDeactivationSuccessMessage`.

<img width="707" alt="Screen Shot 2020-07-23 at 12 50 19 PM" src="https://user-images.githubusercontent.com/28743286/88638861-e9769a00-d070-11ea-83c5-4635f12fd951.png">

Reply:
Are there other places where we're checking for permissions that the `lblInsufficientPermissions` header could be used? Seems like this is a generic label that could be applied across the product wherever we are doing permission checks.

Reply:
The search in codebase does not return any part of code where "Insufficient Permissions" or `Insufficient*` is used so no other code is changed. However, it is a generic label and can be used in future WIs.

Reply:
@melissabarber ^^

Reply:
@RadSFDeveloper Gift Entry uses a similar label, `commonAdminPermissionErrorTitle`, "Check Your Permissions".

For this one, maybe we could change the label name to `commonInsuffientPermissions`

The Gift Entry team has started to use the "common" prefix for anything that could be used generically. There's several other labels like that in customLabels.

@force2b

Reply:
We have commonAdminPermissionErrorTitle "Check Your Permissions" but as Sam pointed out in our side conversations, users can't really check their permissions unless they're admins and "Insufficient Permissions" makes more sense. @ptewson-sfdo we might want to make this part of HoneyGems' cleanup work when our two teams nail down more consistent permission errors messaging.

Reply:
@sam-knox , @melissabarber : the label is renamed to `commonInsufficientPermissions`. Thank you for your feedback.

Reply:
Similarly, the new `lblSelectOptionPlaceholder` containing "Select Option" value is renamed to `commonSelectOptionPlaceholder`.

Reply:
Thanks great, thanks @RadSFDeveloper!